collections/guides/encrypt_us_bank_accounts/index.md

```diff
@@ -0,0 +1,219 @@
+
+In this guide, we will show you how to take an existing API and use Basis Theory to safely store the bank data while and retain the exact application functionality. 
+
+If you'd like to follow along with this guide jump right into the code - <a href="https://github.com/Basis-Theory/basis-theory-js-examples/store-atomic-banks">Find it here!</a>
+
+### Table of contents
+{: .no_toc .text-delta }
+1. 
+{:toc}
+
+## The existing application
+
+Below is an existing nodejs application that is currently storing bank data in memory, this will be used as our foundation as we update to more securely store the bank data. 
+
+_(Note: this is only to simplify the guide, the same process will work when storing in a database)_.
+
+### package.json
+Our simple `package.json`, only has two dependencies to start. [express](https://www.npmjs.com/package/express) allows us to quickly create an HTTP API, and [nodemon](https://www.npmjs.com/package/nodemon) allows us to have our nodejs application restart automatically.
+
+```js
+{
+  "name": "bank-data",
+  "description": "",
+  "version": "1.0.0",
+  "author": "Basis Theory",
+  "scripts": {
+    "start": "nodemon app.js"
+  },
+  "dependencies": {
+    "express": "^4.17.1"
+  },
+  "devDependencies": {
+    "nodemon": "1.18.4"
+  }
+}
+```
+
+### app.json
+Our application simply allows a U.S. Bank Account to be sent into the system, stored in memory, and retrieved back. _(Even though our example doesn't show them, your systems will most likely have authentication, databases, etc.)_
+
+```js
+const express = require('express')
+const app = express()
+app.use(express.json());
+const port = 3000
+
+// simple storage for this example, this is most likely a DB in your system
+let account = {};
+
+// returning the raw bank information
+app.get('/get', async (req, res) => {
+    res.send(account)
+})
+
+// storing the bank information
+app.post('/create', async (req, res) => {
+    const {accountNumber, routingNumber} = req.body;
+
+    account = {accountNumber, routingNumber};
+
+    res.send()
+})
+
+// starting up your API server
+app.listen(port, async () => {
+    console.log(`Example app listening at http://localhost:${port}`)
+})
+```
+
+## Install basis-theory-js
+
+<span class="base-alert warning">
+  <span>
+    To start, you'll need a new Server to Server Application with the <code>token:bank:create</code> and <code>token:bank:read:high</code> permission
+  </span>
+</span>
+
+First, you'll need to install the [basis-theory-js](https://www.npmjs.com/package/@basis-theory/basis-theory-js) npm module
+
+```bash
+    npm install @basis-theory/basis-theory-js --save
+```
+
+## Initialize basis-theory-js 
+
+Next, we will need to initialize `BasisTheory` and store it to be used when banks are being stored:

```

Comment:
```suggestion
Next, we will need to initialize an instance of `BasisTheory` and store a reference to be used later:
```

I also think it reads weird to just call this `BasisTheory` - it would sound better as `an instance of the BasisTheory sdk|library|client` - but I don't know what the right term is that we use elsewhere.